src/utils/supabase/server.test.ts

```
@@ -0,0 +1,268 @@
+      expect.objectContaining({
+        cookies: expect.objectContaining({
+          get: expect.any(Function),
+          set: expect.any(Function),
+        }),
+      })
+    );
+    expect(client).toBe(mockClient);
+  });
+
+  it('should throw error when SUPABASE_URL is missing', () => {
+    delete process.env.NEXT_PUBLIC_SUPABASE_URL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)